install/install.php

```diff
@@ -246,9 +246,14 @@ protected function createDefaultDisplayPreferences() {
       $this->migration->displayMessage("create default display preferences");
 
       // Create standard display preferences
-      $displayprefs = new DisplayPreference();
-      $found_dprefs = $displayprefs->find("`itemtype` = 'PluginFormcreatorFormAnswer'");
-      if (count($found_dprefs) == 0) {
+      $found_dprefs = $DB->request([
```

Comment:
Migration class provides `updateDisplayPrefs`; would not this be more reliable than x queries?

Reply:
indeed, will do